### DISCUSSION TITLE
Possible to change text of a TextLayer?

### DISCUSSION THREAD

**Discussion by vkoopmans-resoluut** [ingrammicro/puzzle-tokens#54] [Ideas]
Hi @MaxBazarov I was wondering if it would be possible to target a text layer and change it's text.
I guess it should be possible right? Do you have an idea how to target the contents of a TextLayer?

**Comment by MaxBazarov** [comment]
Yes. It’s possible.

I just added example here - 
https://github.com/ingrammicro/puzzle-tokens/tree/master/Styles/Set-Text

Please use.

**Comment by vkoopmans-resoluut** [comment]
You are a legend!
That is fantastic!

Thank you,

Vince



Vincent Koopmans
Design Director

0628644396
***@***.***  www.resoluut.com

**Reply by MaxBazarov** [reply]
Thanks!

**Comment by MaxBazarov** [comment]
It’s strange. Your code is valid. 
I will debug it.

**Reply by vkoopmans-resoluut** [reply]
No sorry My bad, I had a symbol name wrong... I changed it while trying :D
It does work!

**Reply by MaxBazarov** [reply]
Yes. 
I changed path to the following to make it working.

#Operating System .iOS .status_bar .white {
    .Carrier {
        pt-text: VirginMedia;
    }
    .Time {
        pt-text: 12:43;
    }
}
